src/bundle/Resources/public/js/scripts/fieldType/base/base-field.js

```diff
@@ -164,6 +165,7 @@
                 nodes.forEach((el) => {
                     const errorNode = this.createErrorNode(validationResult.errorMessage);
 
+                    el.querySelectorAll(SELECTOR_ERROR_CONTAINER).forEach((el) => el.remove());
```

Comment:
This feels like redundant with `existingErrorNodes.forEach((el) => el.remove());`.

In this author field, the problem is that method `findExistingErrorNodes` is not returning all of the error nodes.